models/staging/tpdm/stage/_tpdm__stage.yml

```diff
@@ -12,6 +12,16 @@ referential_integrity_tests:
           to: ref('stg_tpdm__educator_preparation_programs')
           field: k_educator_prep_program
           tags: ['ref_integrity']
+  - k_course_section: &ref_k_course_section
+    - relationships:
+        to: ref('stg_ef3__sections')
+        field: k_course_section
+        tags: ['ref_integrity']
+#  - k_person: &ref_k_person
```

Comment:
you probably do not want to leave commented out lines in the final code, but if this is just temporary while you and Artem are working on things, then it's probably fine.